CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Changes to Calva.
 
 ## [Unreleased]
 
+- Update node version to v14, which is maintenance LTS until 2023-04-30
```

Comment:
You can't leave a blank line between the header and the body here. See How-to-hack wiki page for why (TL;DR: Our CI will croak.)